docs/ext/commands/commands.rst

```diff
@@ -39,10 +39,13 @@ as seen in the example above. The second is using the :func:`~ext.commands.comma
 :meth:`.Bot.add_command` on the instance.
 
 Essentially, these two are equivalent: ::
-
+    import discord
```

Comment:
You removed a newline. I guess that's GitHub's fault for not understanding the intent.